README.md

```diff
@@ -43,7 +43,25 @@
 
 Check our additional materials and our documentation guide [here](https://sap.github.io/cloud-sdk/docs/dzen/getting-started )
 
-## Guidelines on documentation style
+## How we write
+
+- Being concise is a virtue. Write short and on purpose.
+- Provide only relevant context. Make a good assumption of readers' knowledge and make it your starting point for context.
+- Give a link whenever you mention 3rd party resources, library, concept, abbreviation, reference, etc.
+- Avoid abbreviations or introduce them unless it's absolutely obvious and widely adopted.
+- Build upon already available content and use it for reference. Create materials that could be leveraged by others.
+  - Think modular. Extract logically independent parts and link to them from larger scope materials.
+  - Use the same approach you do with coding when structuring your application
```

Comment:
"Use the same approach you do with coding when structuring your application" this is a bit vague to me? What do we use in coding which transfers to writing tutorials?

Reply:
Good question @FrankEssenberger. It's less about Tutorials, indeed, but the larger scale documentation. You guessed a bit the topic of our discussion in Java, where some authors preferred to put instructions to generate a demo App and some basic topics in every article, whilst our convention was to make a separate material on getting you minimal App running and then referring to it so that we can make main material more focused, easy to read and not distracting for side topics if the reader already knows about them.

Coming back to your question, I find a similarity with coding when I extracting independent functional parts from larger materials and then being able to refer to them from other docs that need that reference, tutorial step, etc. This has an analogy with a function call or a nice separation of concerns to make docs a more scalable project. Such modular structure makes it easier to update if smth is outdated not needing to change it in every article duplicating that. Also reminds me of refactoring.

Just saying make it modular should be enough, but some people understand analogies about things they often do better than abstract concepts. I hope it brings a bit more clarity behind my words.

I many people would say it's redundant and doesn't add to mutual alignment - we'll remove it to hydrate our principles to the bare minimum we all commit to.